src/components/Article.jsx

```diff
@@ -1 +1,31 @@
-// export const Article = () => ();
+import React from 'react';
+
+export function Article({
+  title,
+  date,
+  text,
+}) {
+  const today = date;
```

Comment:
You no need 
const today = date;

You already have date :

const dateISO = date.toISOString().slice(0, 10);
  const dateLocaleString = date.toLocaleDateString('en-US', {
    year: 'numeric',
    month: 'long',
    day: 'numeric',
  });